gravitee-am-repository/gravitee-am-repository-jdbc/src/main/resources/liquibase/changelogs/v4_2_27/4.2.27-change-client-column-size.yml

```diff
@@ -0,0 +1,166 @@
+databaseChangeLog:
+  - changeSet:
+      id: 4.2.27-change-client-column-size-remove-indexes
+      author: GraviteeSource Team
+      dbms: mssql
+      changes:
+        # scope_approvals
+        - dropIndex:
+            indexName: idx_scope_approvals_domain_client_user_scope
+            tableName: scope_approvals
+
+        #access_tokens
+        - dropIndex:
+            indexName: idx_access_tokens_client
+            tableName: access_tokens
+        - dropIndex:
+            indexName: idx_access_tokens_domain_client_subject
+            tableName: access_tokens
+
+        #refresh_tokens
+        - dropIndex:
+            indexName: idx_refresh_tokens_client
+            tableName: refresh_tokens
+        - dropIndex:
+            indexName: idx_refresh_tokens_domain_client_subject
+            tableName: refresh_tokens
+
+  - changeSet:
+      id: 4.2.27-change-client-column-size
+      author: GraviteeSource Team
+      changes:
+        #devices
+        - dropIndex:
+            tableName: devices
+            indexName: idx_devices_domain_client_user
+        - dropIndex:
+            tableName: devices
+            indexName: idx_devices_domain_client_user_remember_device_device_id
```

Comment:
**todo**: can you explain in a comment that we are doing the dropIndex (and we recreate them) due to the fact that MySQL has a limited size of indexed value so we are aligning all the RBMDS with the refactored index